.travis.yml

```diff
@@ -83,11 +85,10 @@ jobs:
        channel: stable
     env:
     - SNAPCRAFT_IMAGE_INFO: '{"build_url": "$TRAVIS_BUILD_URL"}'
-    before_install:
+    install:
```

Comment:
required to let the default `before_install` logic set env vars (particularly `$SNAP_CHANNEL`)